example/20_grouped_conv_bwd_weight/grouped_conv_bwd_weight_xdl_bf16.cpp

```diff
@@ -17,8 +17,20 @@ using OutElementOp = PassThrough;
 
 template <ck::index_t NDimSpatial>
 using DeviceConvBwdWeightInstance =
-    ck::tensor_operation::device::DeviceGroupedConvBwdWeightGnwcGkxcGnwk_Xdl_CShuffle<
-        NDimSpatial,          // NDimSpatial
+    ck::tensor_operation::device::DeviceGroupedConvBwdWeight_Xdl_CShuffle<
+        NDimSpatial, // NDimSpatial
```

Comment:
The comment is superfluous.
```suggestion
        NDimSpatial,
```

Reply:
Done.